src/webots/user_commands/WbActionManager.cpp

```diff
@@ -679,26 +676,12 @@ void WbActionManager::populateActions() {
   mActions[GO_TO_LINE] = action;
 
   action = new QAction(this);
-  action->setText(tr("&Toggle Line Comment"));
+  action->setText(tr("&Toggle Line Comment    "));
```

Comment:
Are these spaces necessary?

Reply:
Yes, the padding between the text and the keyboard shortcut are miscalculated, and the easiest solution to prevent this without affecting the other items is to add space

Reply:
In that case, we should add a comment about it.
And maybe exclude the spaces from the translation, e.g., `tr("&Toggle Line Comment") + "      ")`.